<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>961</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_run">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_src">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dst">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
